### PR TITLE
Correct tests by sections

### PR DESCRIPTION
**Pull request by aljones15**
Features:

1. rebased with main
2. adds the logging fix from main
3. add row metadata so tests show up
4. turns data model into a single parameterized test
5. deprecates the older tests

### CODE REVIEW

File: tests/helpers.js [hunk 1]
```diff
@@ -44,6 +44,11 @@ export function setupRow() {
 }
 
 export function getProofs(issuedVc) {
+  // if the implementation failed to issue a VC or sign the VC return
```
Comment (TallTed) [inline tests/helpers.js:47]:
```suggestion
  // if the implementation failed to issue a VC or to sign the VC, return
```

File: tests/80-algorithms-jcs.js [hunk 2]
```diff
@@ -24,7 +25,7 @@ const {match} = endpoints.filterByTag({
 });
 const should = chai.should();
 
-describe('Algorithms - Transformation (eddsa-jcs-2022)', function() {
+describe('eddsa-jcs-2022 - Algorithms - Transformation', function() {
```
Comment (BigBlueHat) [inline tests/80-algorithms-jcs.js:28]: Good move. I like the cryptosuite name first. Thanks!

**Review by PatStLouis — APPROVED** [review]

File: tests/80-algorithms-jcs.js (hunk 2 above)
Comment (aljones15) [reply tests/80-algorithms-jcs.js:28]: it matches the original better, perhaps we should make a style guide and ensure cryptosuite is first? Data Integrity still lists itself first. Also, the sections are nice, but are not necessarily sorted alphabetically in the report which is a little annoying.

**Review by TallTed — COMMENTED** [review]
English lgtm. I cannot judge the code.

**Pull request merged**